src/index.js

```diff
@@ -16,6 +16,7 @@ if (!window.CMS_MANUAL_INIT) {
  * Add extension hooks to global scope.
  */
 if (typeof window !== 'undefined') {
+  window.React = Window.React || React;
```

Comment:
The second `Window` should be lowercase.

Reply:
Oops